src/waveform/renderers/allshader/waveformrendererhsv.cpp

```diff
@@ -0,0 +1,213 @@
+#include "waveform/renderers/allshader/waveformrendererhsv.h"
+
+#include "track/track.h"
+#include "util/colorcomponents.h"
+#include "util/math.h"
+#include "waveform/renderers/allshader/matrixforwidgetgeometry.h"
+#include "waveform/waveform.h"
+#include "waveform/waveformwidgetfactory.h"
+#include "waveform/widgets/allshader/waveformwidget.h"
+#include "widget/wskincolor.h"
+#include "widget/wwidget.h"
+
+using namespace allshader;
+
+WaveformRendererHSV::WaveformRendererHSV(
+        WaveformWidgetRenderer* waveformWidget)
+        : WaveformRendererSignalBase(waveformWidget) {
+}
+
+WaveformRendererHSV::~WaveformRendererHSV() {
+}
```

Comment:
```suggestion
```

Reply:
Yeah, I copy/pasted this from WaveformRendererRGB which also has this (and maybe others too). I will remove all these unnecessary destructors.